### PR TITLE
Update links to resources on modelica.org

### PR DESCRIPTION
**Pull request by t-sommer**

### CODE REVIEW

File: _posts/2017-05-15-fmi-user-meeting-prague.md [hunk 1]
```diff
@@ -5,4 +5,4 @@ layout: post
 ---
 
 You are invited to join us for the next FMI User Meeting in Prague at the Modelica Conference, May 15, 2017.
-You can get the agenda [here](https://www.modelica.org/events/modelica2017/documents/fmi-user-meeting).
+You can get the agenda [here](https://web.archive.org/web/20181116092352/https://www.modelica.org/events/modelica2017/documents/fmi-user-meeting).
```
Comment (beutlich) [inline _posts/2017-05-15-fmi-user-meeting-prague.md:8]:
```suggestion
You can get the agenda [here](https://2017.international.conference.modelica.org/fmi-user-meeting.html).
```

**Review by beutlich — COMMENTED** [review]
Not sure if ready for review since it still is a draft PR.

**Review by chrbertsch — APPROVED** [review]

**Review by chrbertsch — CHANGES_REQUESTED** [review]
There are still some links broken to files that were prevously on SVN.

* [Overview of the FMI, DCP and SSP standards](https://svn.fmi-standard.org/fmi/branches/public/docs/Modelica2019/Industrial_User_Presentations_Modelica2019/001_Overview_FMI_SSP_DCP.pdf)
* [Usage of FMI at Bosch - status and outlook](https://svn.fmi-standard.org/fmi/branches/public/docs/Modelica2019/Industrial_User_Presentations_Modelica2019/2808_Usage_FMI_Bosch.pdf)
* [Co-Simulation scenarios in industrial production plants](https://svn.fmi-standard.org/fmi/branches/public/docs/Modelica2019/Industrial_User_Presentations_Modelica2019/Presentation_CoSimulatedPlants_ModelicaFMI_2019_final.pdf)
* [Enhancing the Model Integration Workflow in Aircraft System Simulation using FMI & SSP](https://svn.fmi-standard.org/fmi/branches/public/docs/Modelica2019/Industrial_User_Presentations_Modelica2019/Saab_Eek_MC2019.pdf)

@t-sommer: Where shall we put the these files? To https://github.com/modelica/fmi-standard.org/tree/master/assets?

Do we have the PDFs any more? @MABackoffice : is there a backup of the SVN?

Some of the PPT files are here https://github.com/modelica/fmi-design/tree/master/Meetings/2019/2019-03-04-FMI-User-Meeting/Presentations but not all. 

We could also merge the PR and fix the remaining issues later

**Comment by chrbertsch** [comment]
Superceeded by #330

**Pull request closed without merging**